### ISSUE TITLE
rpm install opensuse - launcher icon missing

### ISSUE THREAD

**Issue by DrDBanner** [particl/particl-desktop#1712]
**Describe the bug**
The icon in the application launcher is missing after installation.

**To Reproduce**
1. Go to 'application launcher menu'
2. Look for 'Particl Desktop'
3. Launcher there but see the missing icon

**Expected behavior**
A visible launcher icon with Particl Logo.

**Versions (please complete the following information):**
 - OS: [OpenSuse Tumbleweed on Gnome 3.6]
 - Particl Desktop: [e.g. v2.3.2]
**Additional context**
rpm installation via zypper had following output
```
gtk-update-icon-cache: The generated cache was invalid.
warning: %triggerin(gtk3-tools-3.24.13+0-3.3.x86_64) scriptlet failed, exit status 1
```

Have seen the same problem on a fedora installation.

**Comment by DrDBanner** [comment]
This issue seems to break the installation completely on Debian 20.20 reported by a user (not verified)

**Comment by DrDBanner** [comment]
ported from support channel: 
Ubuntu 20.04.3 LTS 64bit  Gnome version 3.36.8 

 got error 
“gtk-update-icon-cache: The generated cache was invalid.
Warning: icon cache generation failed” - 

Particl Desktop icon (logo) doesn’t show but everything else works

**Comment by zaSmilingIdiot** [comment]
Should be resolved as of v3.3.0
